src/Composer.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace TSB\WP\MUPlugin;
+
+/**
+ * Use the necessary namespaces.
+ */
+use Composer\Composer as ComposerComposer;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+use Composer\EventDispatcher\EventSubscriberInterface;
+
+
+class Composer implements PluginInterface, EventSubscriberInterface {
+
+	/**
+	 * Holds the extras array for the root Composer project.
+	 * @var Array
+	 */
+	private $extras;
+
+	/**
+	 * Stores the extras array and config object for later use.
+	 *
+	 * @param Composer    $composer The main Composer object.
+	 * @param IOInterface $io       The I/O Helper object.
+	 * @return void
+	 */
+	public function activate( ComposerComposer $composer, IOInterface $io ) {
+		$this->extras = $composer->getPackage()->getExtra();
+	}
+
+	/**
+	 * Required by PluginInterface: Deactivate the plugin.
+	 *
+	 * @param Composer    $composer The main Composer object.
+	 * @param IOInterface $io       The I/O Helper object.
+	 * @return void
+	 */
+	public function deactivate( ComposerComposer $composer, IOInterface $io ) {
+		// No action needed for this plugin.
+	}
+
+	/**
+	 * Required by PluginInterface: Uninstall the plugin.
+	 *
+	 * @param Composer    $composer The main Composer object.
+	 * @param IOInterface $io       The I/O Helper object.
+	 * @return void
+	 */
+	public function uninstall( ComposerComposer $composer, IOInterface $io ) {
+		// No action needed for this plugin.
+	}
+	/**
+	 * Subscribes to autoload dump and package install events.
+	 *
+	 * When `pre-autoload-dump` fires, run the `dumpRequireFile` method.
+	 * When `pre-package-install` fires, run the `overridePluginTypes` method.
+	 *
+	 * @return array The event subscription map.
+	 */
+	public static function getSubscribedEvents() {
+		return array(
+			'post-package-install'  => 'copyMUPlugins',
+			'post-package-update'   => 'copyMUPlugins',
+			'pre-package-uninstall' => 'deleteMUPlugins',
+		);
+	}
+
+	public function copyMUPlugins() : void {
+		echo "Copying 26b/wp-must-use mu-plugins...\n";
+		$local_plugins_path = dirname( __DIR__ ) . '/plugins/';
```

Comment:
This should be a must-use, no need to enable. We could have a filter here, later on, so it can be simple to eject some plugins to be controlled as activatable or not.
```suggestion
		$local_plugins_path = dirname( __DIR__ ) . '/mu-plugins/';
```